src/classes/TriggerHandler.cls

```diff
@@ -74,16 +88,55 @@ public virtual class TriggerHandler {
     TriggerHandler.bypassedHandlers.add(handlerName);
   }
 
+  public static void bypass(List<String> handlersNames) {
+    TriggerHandler.bypassedHandlers.addAll(handlersNames);
+  }
+
+  public static void bypass(Type handlerType) {
+    TriggerHandler.bypass(handlerType.getName());
+  }
+
+  public static void bypassAll() {
+    globalBypass = true;
+  }
+
   public static void clearBypass(String handlerName) {
     TriggerHandler.bypassedHandlers.remove(handlerName);
   }
 
+  public static void clearBypass(Type handlerType) {
+    TriggerHandler.clearBypass(handlerType.getName());
+  }
+
+  public static void clearBypass(List<String> handlersNames) {
+    TriggerHandler.bypassedHandlers.removeAll(handlersNames);
+  }
+
   public static Boolean isBypassed(String handlerName) {
-    return TriggerHandler.bypassedHandlers.contains(handlerName);
+    return (globalBypass || TriggerHandler.bypassedHandlers.contains(handlerName));
+  }
+
+  public static Boolean isBypassed(Type handlerType) {
+    return (globalBypass || TriggerHandler.bypassedHandlers.contains(handlerType.getName()));
   }
 
   public static void clearAllBypasses() {
-    TriggerHandler.bypassedHandlers.clear();
+    if(globalBypass) {
+      globalBypass = false;
+    } else {
+      TriggerHandler.bypassedHandlers.clear();
+    }
+  }
+
+  public static void showLimits() {
+    showLimits = true;
+  }
```

Comment:
Overloaded to allow for calling method with a boolean. This allows toggling off.

```suggestion
  public static void showLimits() {
    showLimits(true);
  }
  
  public static void showLimits(Boolean enabled) {
    showLimits = enabled;
  }
```